### ISSUE TITLE
Same quantities are not comparable

### ISSUE THREAD

**Issue by Azbesciak** [unitsofmeasurement/uom-se#212]
Hello,
I just upgraded from 0.8 to 1.0, and have some problem with quantities (it occured also before, but without exception).
Lets consider following code:
````java
import org.junit.jupiter.api.Test;
import tec.uom.se.ComparableQuantity;
import tec.uom.se.quantity.Quantities;

import javax.measure.quantity.Speed;

import static org.junit.jupiter.api.Assertions.assertTrue;
import static tec.uom.se.unit.Units.KILOMETRE_PER_HOUR;

class FailTest {
    @Test()
    void test() {
        var quantity = (ComparableQuantity<Speed>) Quantities.getQuantity("100 km/h");
        var speedFromInt = Quantities.getQuantity(100, KILOMETRE_PER_HOUR);
        assertTrue(quantity.isEquivalentTo(speedFromInt));
    }
}
````
This should pass, but it doesn't. It may be connected with 
https://github.com/unitsofmeasurement/uom-systems/issues/132 - in runtime I also receive following exception:
````
javax.measure.UnconvertibleException: javax.measure.IncommensurableException: km/h is not compatible with km/h
	at tec.uom.se.AbstractUnit.getConverterTo(AbstractUnit.java:296) ~[uom-se-1.0.10.jar:na]
	at tec.uom.se.AbstractQuantity.to(AbstractQuantity.java:181) ~[uom-se-1.0.10.jar:na]
	at tec.uom.se.AbstractQuantity.to(AbstractQuantity.java:110) ~[uom-se-1.0.10.jar:na]
````
And indeed - during debug I get something like comparing units `[L]/[T]` with `[T]/([L]*[M])` [AbstractUnit](https://github.com/unitsofmeasurement/uom-se/blob/f78a26e68e07000150fc72259db96cd01861cc33/src/main/java/tec/uom/se/AbstractUnit.java#L219):
![image](https://user-images.githubusercontent.com/17334466/58469887-c4bb2580-8140-11e9-82d2-24934005cbfd.png)

**Comment by andi-huber** [comment]
Running above test against latest Indriya's 'master' branch does not fail:

(using uom-lib-common:2.1-SNAPSHOT)

**Comment by keilw** [comment]
Yes it works with 2.x, I created a very minimalistic fix in uom-se that is not meant as a full replacement for the new functionality in 2.x but if some apps or users were unable to upgrade to the new releases. Also added `getAsDouble()` and `DoubleSupplier` to the children of `MultiplyConverter` in Indriya for consistency. There it's mainly a convenience method. 
I only saw `MultiplyConverter` (the old version in uom-se) and `RationalConverter` to be comparable, the others are usually different and have no factor that could be compared in a meaningful way.
